src/app/project/FeatureDetails.js

```diff
@@ -146,6 +182,28 @@ define([
                 });
                 this.newFeatureWizard = wizard;
             }
+        },
+        onDeleteFeatureClick: function () {
+            // summary:
+            //      user has clicked the delete feature button
+            console.log('app.project.FeatureDetails:onDeleteFeatureClick', arguments);
+
+            var that = this;
+            var projectId = router.getProjectId();
+            var onErrorWithDefault = lang.partial(onError, 'Error deleting feature!');
+            xhr.del(config.urls.api + '/project/' + projectId + '/feature/delete/' + that.currentId, {
```

Comment:
I would remove the delete fragment and allow the verb to dictate what is happening. I think it should be the same signature as the feature details with the query string category